Disable version check. This is needed so that downstream packages which may bundle different versions of
the esbuild client library can be built at all.

--- a/cmd/esbuild/main.go
+++ b/cmd/esbuild/main.go
@@ -189,7 +189,6 @@
 				logger.PrintErrorToStderr(osArgs,
 					fmt.Sprintf("Cannot start service: Host version %q does not match binary version %q",
 						hostVersion, esbuildVersion))
-				os.Exit(1)
 			}
 
 		case strings.HasPrefix(arg, "--ping"):
